### DISCUSSION TITLE
Qustion about unloading trailler

### DISCUSSION THREAD

**Discussion by Vortexgamin** [Courseplay/Courseplay_FS22#2145] [Q&A]
I trying to unload combine using trailer.  First field worked great, combined with autodrive.  When move to second field and try to set up the same way, I get this "CP: Unloader too far away from field".  Both harvester and trailer on same field.  Not sure what I am doing wrong.  It was doing in the older version and updated to new version with same.   Any help would be appreciated

![CP](https://user-images.githubusercontent.com/106041480/204160751-02c96385-58f3-4fec-b2d2-150248db6b20.png)

**Comment by Tensuko** [comment]
Looks like the field marker is not on that field. Use the target icon in the HUD.
